ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -607,6 +608,7 @@ objects:
               fi
               sleep 1
             done
+            echo "Google's OAuth API is available"
```

Comment:
Kill this

Reply:
> Kill this

Done with 4759620e0 -> 3ef39054f, but this means that we don't have a failsafe if we miss something else and die here without logging an explanation.